doc/historylog.rst

```diff
@@ -2,26 +2,49 @@
 History
 #######
 
-With the **History** pane, you can view an automatically de-duplicated, time-stamped list of every command you enter into any connected :doc:`ipythonconsole`.
+With the **History** pane, you can view all the commands you've entered into any :doc:`ipythonconsole`, along with their timestamp.
 
 .. image:: images/history/history-menu.png
    :alt: Spyder History Log, displaying a list of previously executed commands
 
+(Replace screenshot for screenshot without the options menu)
 
 
-=====================
-Using the History Log
-=====================
 
-Navigating the :guilabel:`History Log` is very straightforward.
-Each session is marked by a date and time-stamp, making it easy to remember when you executed a certain command.
+======================
+Using the History pane
+======================
+
+Navigating the :guilabel:`History` pane is very straightforward.
+Each Spyder session is marked by a date and timestamp, making it easy to remember when you executed a certain command.
 Statements can be selected and copied from the context menu or with the normal system shortcuts.
-Just like in the editor, highlighting a word or phrase displays all other occurrences, and full syntax highlighting is also supported.
-Finally, the top-right options menu (:guilabel:`Gear` icon) allows you to toggle soft-wrapping of long lines (:guilabel:`Wrap lines`), and set the number of commands the :guilabel:`History Log` should remember (:guilabel:`History`).
+Just like in the Editor, selecting a word or phrase displays all other occurrences, and full syntax highlighting is also supported.
+
+
+
+============
+Options Menu
+============
+
+The top-right options menu (:guilabel:`Hamburger` icon) allows you to set the number of commands that the :guilabel:`History` pane should remember.
+
+(Gif with dialog for changing the maximum entries)
 
-The :guilabel:`History Log` is stored in the :file:`.spyder-py3` (Python 3) or :file:`spyder` (Python 2) directory in your user home folder (by default, :file:`C:/Users/{username}` on Windows, :file:`/Users/{username}` for macOS, and typically :file:`/home/{username}` on GNU/Linux).
+You can also toggle wrapping of long lines (:guilabel:`Wrap lines`), and whether the line number is displayed to the left of the text (:guilabel:`Show line numbers`).
+
+(Gif to display deactivating wrapping and displaying line number)
+
+
+
+==============
+Advanced usage
+==============
+
+The list of commands shown in the :guilabel:`History` pane are stored in :file:`history.py` in the :file:`.spyder-py3` directory in your user home folder (by default, :file:`C:/Users/{username}` on Windows, :file:`/Users/{username}` for macOS, and typically :file:`/home/{username}` on GNU/Linux).
```

Comment:
Using `/` for paths consistently across platforms is what is specified in the style guide and implemented across the current docs for both [paths in general](https://github.com/spyder-ide/spyder-docs/blob/master/STYLEGUIDE.md#writing-all-languages) and the [the file role specifically](https://github.com/spyder-ide/spyder-docs/blob/master/STYLEGUIDE.md#rest-style-elements). Given most paths mentioned in the docs are not platform-specific, it makes sense to just pick one path separator and use it consistently, rather than treating specially the small number of absolute paths that are unique to Windows.

Furthermore, in Python source code `/` is used almost universally for paths regardless of platform (by convention with string paths, and by requirement with `pathlib` `Path`s); indeed, our own copy relative/absolute path commands copy them this way, so it will avoid things breaking things if users copy them into their code, the console. Finally, given this usage, users are users across platforms are sufficiently used to `/` in paths in Python code, URLs and elsewhere that the likelihood of confusion by using `/` on Windows is effectively zero; despite many places in Spyder's UI where `/` is or was used on Windows, the only cases where this has genuinely caused an issue for even beginners was when `/` and `\` were mixed in the _same_ path or otherwise used inconsistently, which this would help avoid.

Reply:
> Using / for paths consistently across platforms is what is specified in the style guide

Ok, I wasn't aware of that. Then we can leave it as it is.